http/graphql-client/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>graphql-client</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: HTTP: GraphQL Client</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-graphql</artifactId>
+            <scope>provided</scope>
```

Comment:
Why this provided scope? is it needed?

Reply:
It is require for a hack in http/graphql-client/src/test/java/io/quarkus/ts/http/graphql/SeparateDependenciesIT.java to work.
We need to have this dependency, since otherwise this module will not compile *before* tests are run.
We can not have this dependency in "compile" scope, because it would be added into the application under test and the issue will not be reproducible.